apps/meteor/tests/unit/app/lib/server/functions/validateUsername.spec.ts

```diff
@@ -0,0 +1,95 @@
+import { expect } from 'chai';
+import proxyquire from 'proxyquire';
+
+const proxySettings = {
+	settings: {
+		get: (key: string): string | null => {
+			if (key === 'UTF8_User_Names_Validation') {
+				// Default value set at apps/meteor/server/settings/general.ts
+				return '[0-9a-zA-Z-_.]+';
+			}
+			return null;
```

Comment:
You can declare a stub for this (with sinon) and then use `stub.withArgs('UTF...').returns('')`